cmd/build/main.go

```diff
@@ -146,7 +146,9 @@ func processResults(waitGroup *sync.WaitGroup, results chan *TileResponse, proce
 
 func main() {
 	urlTemplateStr := flag.String("url", "", "URL template to make tile requests with.")
-	outputStr := flag.String("output", "", "Path to output mbtiles file.")
+	outputMode := flag.String("mode", "mbtiles", "Valid modes are: disk, mbtiles.")
+	outputStr := flag.String("output", "", "Path, or DSN string, to output files. DEPRECATED - please use -dsn instead.")
```

Comment:
I think we probably don't need to keep the old option around. This code is new enough that we're not breaking any promises.

Reply:
Updated in https://github.com/sfomuseum/go-tilepacks/commit/eedc3b515ceac0a3cbbf69c7a118302515a11a86

Reply:
Great, thanks!